src/components/Equipment/ContentScuba.js

```diff
@@ -0,0 +1,158 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  withStyles,
+  ButtonBase,
+  Typography,
+  Grid,
+  ClickAwayListener
+} from '@material-ui/core';
+
+import equipmentImage from '../../assets/equipment.jpg';
+import close from '../../assets/icons/hot-spot-3.svg';
+import search from '../../assets/icons/plus-2.svg';
+import battery from '../../assets/mgl-0768.png';
+
+const styles = theme => ({
+  hidden: {
+    display: 'none !important'
+  },
+  details: {
+    zIndex: 1,
+    position: 'absolute',
+    left: '35%',
+    display: 'flex',
+    alignSelf: 'center',
+    height: '35.25rem',
+    width: '21.875rem',
+    overflow: 'hidden',
+    '&:before': {
+      content: '""',
+      background: `url(${equipmentImage})`,
+      backgroundSize: 'cover',
+      position: 'absolute',
+      left: '-1.5625rem',
+      top: '-1.5625rem',
+      right: 0,
+      bottom: 0,
+      boxShadow: 'inset 0 0 0 187.5rem rgba(255,255,255,0.3)',
+      filter: 'blur(0.625rem)'
+    },
+    [theme.breakpoints.up('sm')]: {
+      left: '10%',
+      height: '35.25rem'
+    },
+    [theme.breakpoints.up('md')]: {
+      left: '10%',
+      height: '31.25rem',
+      width: '28.125rem'
+    },
+    [theme.breakpoints.up('lg')]: {
+      height: '40.625rem'
+    }
+  },
+  content: {
```

Comment:
Since you refactored, change `content` to `root`and move it to the top.

and `<div className={classes.content}>` should then be `<div className={classes.root}>`